pylint/checkers/refactoring/recommendation_checker.py

```diff
@@ -35,6 +35,16 @@ class RecommendationChecker(checkers.BaseChecker):
             "Both the key and value can be accessed by iterating using the .items() "
             "method of the dictionary instead.",
         ),
+        "C0207": (
+            "Consider using %s instead",
+            "consider-using-str-partition",
+            "Emitted when accessing only the first or last element of a str.split(sep). "
+            "Or when a str.split(sep,maxsplit=1) is used. "
+            "The first and last element can be accessed by using str.partition(sep)[0] "
+            "or str.rpartition(sep)[-1] instead, which is less computationally "
+            "expensive and works the same as str.split() or str.rsplit() with a maxsplit "
+            "of 1",
```

Comment:
```suggestion
            "Emitted when accessing only the first or last element of str.split(). "
            "The first and last element can be accessed by using str.partition()[0] "
            "or str.rpartition()[-1] instead."
```
Again, we probably don't need to overcomplicate it.

Reply:
resolved cd86ca0